### PR TITLE
CLI change messages to i18n

### PR DESCRIPTION
**Pull request by phash**
Using MessagesUtil.get(..) for messages on the CLI

### CODE REVIEW

**Comment by semuxproject** [comment]
I will fix the Scrutinizer issues. 

@cryptokat  your opinion on CLI localization?

File: src/main/resources/org/semux/gui/messages.properties [hunk 1]
```diff
@@ -158,3 +158,27 @@ AddAddress = Add an address
 
 BackupFileExists = "{0}" already exists. Do you want to replace it?
 
+#cli
```
Comment (cryptokat) [inline src/main/resources/org/semux/gui/messages.properties]: I think it will be more appropriate to move these CLI messages into an independent file because they are not related to the GUI package: `src/main/resources/org/semux/cli/messages.properties`

**Comment by cryptokat** [comment]
Hi @phash 
FYI: I am working on improvements based on your branch in #281 in order to resolve some reported issues from Scrutinizer
